### ISSUE TITLE
Add Sharing

### ISSUE THREAD

**Issue by acekyd** [acekyd/Random48LawsOfPower#4]
Share to Twitter and Facebook

**Comment by dapseen** [comment]
Good to one to look into

**Comment by phavor** [comment]
@acekyd I do the adding of the button

**Comment by acekyd** [comment]
No, that would not be necessary at all for adding social sharing buttons. @phavor

**Comment by phavor** [comment]
Then it means I've totally misconstrued 

> Don't forget to share the content of the law and also add link to the extension in the sharing.
